src/marshmallow/fields.py

```diff
@@ -606,9 +607,12 @@ def _bind_to_schema(self, field_name, schema):
     def _serialize(self, value, attr, obj, **kwargs):
         if value is None:
             return None
-        if utils.is_collection(value):
-            return [self.inner._serialize(each, attr, obj, **kwargs) for each in value]
-        return [self.inner._serialize(value, attr, obj, **kwargs)]
+        if not utils.is_collection(value):
+            value = [value]
```

Comment:
Unrelated to this PR but I'm wondering the point of supporting this.

Removing line 611 in original code only breaks the test testing specifically this:

```py
class DateTimeList:
    def __init__(self, dtimes):
        self.dtimes = dtimes

def test_datetime_list_serialize_single_value(self):
    obj = DateTimeList(dt.datetime.utcnow())
    field = fields.List(fields.DateTime)
    result = field.serialize("dtimes", obj)
    assert len(result) == 1
    assert type(result[0]) == str
```